tools/setup/emoji/emoji_names.py

```diff
@@ -32,8 +34,279 @@
     "1f198": {"canonical_name": "sos", "aliases": []},
     "1f199": {"canonical_name": "squared_up", "aliases": []},
     "1f19a": {"canonical_name": "vs", "aliases": []},
+    "1f1e6-1f1e8": {"canonical_name": "flag_ascension_island", "aliases": []},
+    "1f1e6-1f1e9": {"canonical_name": "flag_andorra", "aliases": []},
+    "1f1e6-1f1ea": {"canonical_name": "flag_united_arab_emirates", "aliases": []},
+    "1f1e6-1f1eb": {"canonical_name": "flag_afghanistan", "aliases": []},
+    "1f1e6-1f1ec": {"canonical_name": "flag_antigua_and_barbuda", "aliases": []},
+    "1f1e6-1f1ee": {"canonical_name": "flag_anguilla", "aliases": []},
+    "1f1e6-1f1f1": {"canonical_name": "flag_albania", "aliases": []},
+    "1f1e6-1f1f2": {"canonical_name": "flag_armenia", "aliases": []},
+    "1f1e6-1f1f4": {"canonical_name": "flag_angola", "aliases": []},
+    "1f1e6-1f1f6": {"canonical_name": "flag_antarctica", "aliases": []},
+    "1f1e6-1f1f7": {"canonical_name": "flag_argentina", "aliases": []},
+    "1f1e6-1f1f8": {"canonical_name": "flag_american_samoa", "aliases": []},
+    "1f1e6-1f1f9": {"canonical_name": "flag_austria", "aliases": []},
+    "1f1e6-1f1fa": {"canonical_name": "flag_australia", "aliases": []},
+    "1f1e6-1f1fc": {"canonical_name": "flag_aruba", "aliases": []},
+    "1f1e6-1f1fd": {"canonical_name": "flag_\u00e5land_islands", "aliases": ["flag_aland_islands"]},
```

Comment:
This file would be more readable without the `\u` escapes, which is easy to fix:

```diff
--- a/tools/setup/emoji/generate_emoji_names
+++ b/tools/setup/emoji/generate_emoji_names
@@ -82,7 +82,7 @@ def main() -> None:
             "EMOJI_NAME_MAPS: Dict[str, Dict[str, Any]] = {\n"
         )
         for (key, emoji_names) in all_emojis.items():
-            f.write(f'    "{key}": {json.dumps(emoji_names)},\n')
+            f.write(f"    {key!r}: {emoji_names!r},\n")
         f.write("}\n")
 
     print(
```

Reply:
great, thanks!